sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_file_client_async.py

```diff
@@ -146,6 +146,10 @@ class ShareFileClient(AsyncStorageAccountHostsMixin, ShareFileClientBase):
     :keyword str secondary_hostname:
         The hostname of the secondary endpoint.
     :keyword int max_range_size: The maximum range size used for a file upload. Defaults to 4*1024*1024.
+    :keyword str audience: The audience to use when requesting tokens for Azure Active Directory authentication.
+        Only has an effect when credential is of type TokenCredential. Specify your Storage Account name to use
+        the https://account.blob.core.windows.net audience. Otherwise, if not specified, the default audience
+        of https://storage.azure.com/ will be used.
```

Comment:
Missed this one too. Maybe just check all of them in case I missed one.

Reply:
Good catch, found the remaining 3 (2 you identified, last one was hiding in _path_client_async. I think I searched for this in VSCode but left it pointed at a certain package and so I thought I was done 😄